tests/test_project.py

```diff
@@ -416,14 +416,14 @@ def test_grep(west_init_tmpdir):
 
     assert re.search('west-commands', cmd('grep -- -- -commands'))
 
-
-def test_update_projects(west_init_tmpdir):
+@pytest.mark.parametrize("options", ["", "-j 1", "-j 2", "-j"])
```

Comment:
is `-j` indented? should not be `-j -1`

Reply:
I agree seems so from here: `parser.add_argument('-j', '--jobs', nargs='?', const=-1,` that `-1` should be passed